source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRMediaRenderer.mm

```diff
@@ -156,7 +157,8 @@ - (UIView *)render:(UIView<ACRIContentHoldingView> *)viewGroup
     }
 
     if (mediaElem->GetHeight() == HeightType::Stretch) {
-        [viewGroup addArrangedSubview:[viewGroup addPaddingFor:contentholdingview]];
+        NSString *areaName = [NSString stringWithCString:elem->GetAreaGridName()->c_str() encoding:NSUTF8StringEncoding];
+        [viewGroup addArrangedSubview:[viewGroup addPaddingFor:contentholdingview] withAreaName:areaName];
     }
 
```

Comment:
here and everywhere let's follow the objective c style - open braces on new line.